tools/serve/serve.py

```diff
@@ -40,6 +40,8 @@ class WrapperHandler(object):
 
     __meta__ = abc.ABCMeta
 
+    headers = ()
```

Comment:
Sure thing. Why btw? Just for my own learning, I haven't done python in a while.